app/services/account_reset/grant_requests_and_send_emails.rb

```diff
@@ -48,5 +49,17 @@ def grant_request_and_send_email(arr)
       end
       true
     end
+
+    def fraud_wait_period_not_met?(arr, now)
```

Comment:
The `?` suffix implies this returns a boolean, but it only returns a boolean in the body of the `if` condition, and otherwise would return `nil`. I don't think this has a meaningful impact as-is.